hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -785,24 +786,24 @@ public OzoneBucket getBucketDetails(
         volumeName, prevBucket, bucketPrefix, maxListResult);
 
     return buckets.stream().map(bucket -> new OzoneBucket(
-        conf,
```

Comment:
@JyotinderSingh can we revert RPCclient.java changes. It looks to me there is no change done except indentation.

Reply:
Yes, it must have been introduced during an intermediate change. I'll revert this. Thanks!